coursemapper-kg/recommendation/app/services/course_materials/recommendation/resource_recommender.py

```
@@ -1,11 +1,18 @@
-import concurrent.futures
-
 from ..db.neo4_db import NeoDataBase
 
 from .recommendation_type import RecommendationType
 from .recommender import Recommender
+import numpy as np
 from config import Config
 
+import numpy as np
```

Ruff (0.12.2)

Redefinition of unused `np` from line 5

Remove definition: `np`

(F811)